internal/pkg/cli/local_run.go

```diff
@@ -56,8 +67,32 @@ func (o *localRunOpts) Validate() error {
 
 // Ask prompts the user for any unprovided required fields and validates them.
 func (o *localRunOpts) Ask() error {
-	//TODO(varun359): Validate and Ask SvcEnvName
+	if err := o.validateAndAskWkldEnvName(); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
If we don't have more prompts to add:
```suggestion
	return  o.validateAndAskWkldEnvName()
```